internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -62,6 +63,51 @@ message GetOperationsResponse {
   repeated Operation operations = 1;
 }
 
+message TraceQueryParams {
```

Comment:
@yurishkuro Do we want to mark any of these fields as required?

Reply:
We could, but in api-v3